### ISSUE TITLE
Lutris no longer quits after closing (after all running games are closed

### ISSUE THREAD

**Issue by Fxzzi** [lutris/lutris#5488]
### Bug description

If I remember correctly, quite a few updates ago Lutris used to quit fully from my tray just from clicking the X after all games are closed. However now, lutris for some reason stays open in my tray, meaning I have to manually Ctrl+Q or quit from tray icon. Am I missing an option for this orrr?

### How to Reproduce

- Open Lutris
- Close lutris with X
- Observe lutris still open in tray

### Expected behavior

- Open lutris
- Close lutris with X
- Lutris is not in tray, and no lutris process is running

### Log output

```shell
N/A
```

### System Information

```shell
Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: NO
Mangohud: YES
Gamemode: YES
Steam: YES
In Flatpak: NO
[System]
OS: Arch Linux rolling n/a
Arch: x86_64
Kernel: 6.9.2-zen1-1.1-zen
Desktop: Hyprland
Display Server: wayland
[CPU]
Vendor: AuthenticAMD
Model: AMD Ryzen 5 5600X 6-Core Processor
Physical cores: 6
Logical cores: 12
[Memory]
RAM: 31.3 GB
Swap: 16.0 GB
[Graphics]
Vendor: NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce RTX 3070/PCIe/SSE2
OpenGL Version: 4.6.0 NVIDIA 555.42.02
OpenGL Core: 4.6.0 NVIDIA 555.42.02
OpenGL ES: OpenGL ES 3.2 NVIDIA 555.42.02
Vulkan Version: 1.3.285
Vulkan Drivers: NVIDIA GeForce RTX 3070 (1.3.278)
```

**Comment by danieljohnson2** [comment]
Wasn't that a bug in the first place? What's the point of having Lutris in the tray in the first place, if it won't stay there when the main Lutris window is closed?

**Comment by Fxzzi** [comment]
Yeah I get that, thanks for the explanation. I've disabled the tray icon to restore my expected behaviour. Is there any way to close lutris once your game has launched / quit?

**Comment by danieljohnson2** [comment]
There's the "Minimize client when game is launched" option, which isn't really the same thing.

I think if you create a desktop shortcut for a game from Lutris, that shortcut should not keep Lutris running after the game exits.
